apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/UpdaterMediatorImpl.java

```diff
@@ -52,13 +54,15 @@ public void shutdownApplication() {
 
     @Override
     public void suspendBlockchain() {
+        lookupDexOrderProcessor().suspendContractProcessor();
```

Comment:
Better to emit event to disable dexOrder processor (do not allow coupling between updater and exchange)

Reply:
done